Source/Charts/Data/Implementations/Standard/ChartData.swift

```diff
@@ -75,256 +65,120 @@ open class ChartData: NSObject, ExpressibleByArrayLiteral
     /// calc minimum and maximum y value over all datasets
     @objc open func calcMinMax()
     {
-        _yMax = -Double.greatestFiniteMagnitude
-        _yMin = Double.greatestFiniteMagnitude
-        _xMax = -Double.greatestFiniteMagnitude
-        _xMin = Double.greatestFiniteMagnitude
-        
-        forEach { calcMinMax(dataSet: $0) }
-        
-        _leftAxisMax = -Double.greatestFiniteMagnitude
-        _leftAxisMin = Double.greatestFiniteMagnitude
-        _rightAxisMax = -Double.greatestFiniteMagnitude
-        _rightAxisMin = Double.greatestFiniteMagnitude
-        
-        // left axis
-        let firstLeft = getFirstLeft(dataSets: dataSets)
```

Comment:
it's weird we have this logic. I checked it's from 3dbd57b. seems indeed duplicated about `getFirstLeft`. good to see we finally fix it